src/libraries/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -80,6 +81,82 @@ private void AssertNonZeroAllZeroDarwin(long value)
             }
         }
 
+        public static IEnumerable<object[]> SignalTestData()
+        {
+            if (OperatingSystem.IsWindows())
+            {
+                // GenerateConsoleCtrlEvent only supports sending CTRL_C_EVENT and CTRL_BREAK_EVENT
+                yield return new object[] { PosixSignal.SIGINT };
+                yield return new object[] { PosixSignal.SIGQUIT };
+            }
+            else
+            {
+                foreach (PosixSignal signal in Enum.GetValues<PosixSignal>())
+                {
+                    yield return new object[] { signal };
+                }
+                // Test a few raw signals.
+                yield return new object[] { (PosixSignal)3 }; // SIGQUIT
```

Comment:
Why not use `PosixSignal.SIGQUIT` instead? Is it because `Interop.Sys.GetPlatformSignalNumber` expects a different value ?